runtime/onert/core/include/util/ShapeInference.h

```diff
@@ -72,7 +72,8 @@ ir::Shape inferMaxPoolShape(const ir::Shape &in_shape, const ir::operation::MaxP
 
 // TODO write op starting from N
 // TODO write op starting from P
-// TODO write op starting from R
+ir::Shape inferReduceShapes(const ir::Shape &input_shape, const std::vector<int> &axes,
+                            bool keep_dims);
```

Comment:
@YiHyunJin Could you please put the definition of `inferReduceShapes` in `ShapeInference.cc`? It seems to use the definition in `ReduceSum.cc`.